File: inc/commands/biem.py
from inc.classes.Command import Command
from inc.classes.PlaySound import PlaySound

class myCommand(Command):

    COMMAND = '/biem'
    DESCRIPTION = 'BIEM MENEER'

    def run(self, data):
        self.bot.sendMessage(data['chat_id'], "BIEM MENEER!!")
        PlaySound().play('BIEM.mp3')

File: apps/command-server/commands/user/biem.js
import Command from '../base.js'

class Biem extends Command{
    constructor(){
        super()
        this.command = 'biem'
        this.capabilitiesLevel = 1
        this.description = 'Biem'
    }
    async run(){
        this.play('biem.mp3')
        await this.say('Biem!!!!!!!')
    }
}

export default Biem